internal/schema-registry/command_dek_create.go

```diff
@@ -0,0 +1,92 @@
+package schemaregistry
+
+import (
+	"github.com/spf13/cobra"
+
+	srsdk "github.com/confluentinc/schema-registry-sdk-go"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/config"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *command) newDekCreateCommand(cfg *config.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create a DEK.",
+		Args:  cobra.NoArgs,
+		RunE:  c.dekCreate,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Create a DEK with KEK "test", and subject "test-value":`,
+				Code: "confluent schema-registry dek create --name test --subject test-value --version 1",
+			},
+		),
+	}
+
+	cmd.Flags().String("name", "", "Name of the KEK.")
+	cmd.Flags().String("subject", "", "Subject of the DEK.")
+	pcmd.AddAlgorithmFlag(cmd)
+	cmd.Flags().Int32("version", 1, "Version of the DEK.")
```

Comment:
Since the `--version` flag is required, it should not have a default value since that would be misleading to users.
```suggestion
	cmd.Flags().Int32("version", 0, "Version of the DEK.")
```